src/Avalonia.Input/IMouseDevice.cs

```diff
@@ -13,8 +13,10 @@ public interface IMouseDevice : IPointerDevice
         [Obsolete("Use PointerEventArgs.GetPosition")]
         PixelPoint Position { get; }
 
+        [Obsolete]
         void TopLevelClosed(IInputRoot root);
```

Comment:
Not used anymore. TopLevel can now clear pointerover state without access to the mouse device

Reply:
Since Obsolete are being removed for 11.0, should you just remove this now? Is it time to start breaking master?

Reply:
Ah, nevermind. I see your comments below about another PR to break things.

Reply:
Yes, it's fine to break master, but we don't know yet for sure if customers will ask to backport pen PR or not.
So it's only about this specific feature and related.